lib/karen.coffee

```diff
@@ -117,7 +117,14 @@ class MockNode extends MockElement
   getAttribute: (name) ->
     @attributes[name]
 
-  getElementsByTagName: -> []
+  getElementsByTagName: (name) ->
+    found = []
+    for child in @children
+      if child.tagName.toLowerCase() == name.toLowerCase()
+        found.push(child)
+      if child.tagName.toLowerCase() != 'iframe'
```

Comment:
Since html is case insensitive `<a> == <A>` it is reasonable to assume that for a programmatic reflection api that there is a convention on casing. I could not find a reference but I have yet to discover a `tagName` in anything other than uppercase.